src/policykitagent.h

```diff
@@ -65,7 +65,7 @@ public slots:
 
     void request(const QString &request, bool echo);
     void completed(bool gainedAuthorization);
-    void showError(const QString &text);
+    static void showError(const QString &text);
```

Comment:
IMHO, the above line is the opposite of improving readability.

Reply:
In this context it's not important. Removed.